test/persistence-hooks.suite.js

```diff
@@ -1529,7 +1529,7 @@ module.exports = function(dataSource, should, connectorCapabilities) {
           if (err) return done(err);
 
           ctxRecorder.records.should.eql(aCtxForModel(TestModel, {
-            data: {id: existingInstance.id, name: 'changed'},
+            data: {name: 'changed'},
```

Comment:
This test was incorrectly changed by #1598, I am reverting that change now that the implementation has been fixed.

See https://github.com/strongloop/loopback-datasource-juggler/pull/1598/files#diff-145ab49855b38984ba190919e6e7397dL1508